frontend/src/components/pages/Dropoff/components/DropoffCard.tsx

```diff
@@ -0,0 +1,113 @@
+import { StarIcon, TimeIcon } from "@chakra-ui/icons";
+import {
+  Badge,
+  Box,
+  HStack,
+  IconButton,
+  Menu,
+  MenuButton,
+  MenuItem,
+  MenuList,
+  Text,
+  useDisclosure,
+  VStack,
+} from "@chakra-ui/react";
+import React from "react";
+import { useHistory } from "react-router-dom";
+
+import { EllipsisIcon } from "../../../common/icons";
+import { SchedulingFormProps } from "../../Scheduling/types";
+import DeleteScheduleModal from "./DeleteScheduleModal";
+
+interface DropoffCardProps {
+  schedule: SchedulingFormProps;
+  onDelete: () => void;
+}
+
+const DropoffCard = ({ schedule, onDelete }: DropoffCardProps): JSX.Element => {
+  const startDate = new Date(schedule.startTime).toDateString();
+  const startTime = new Date(schedule.startTime).toLocaleTimeString();
+  const history = useHistory();
+
+  const { isOpen, onOpen, onClose } = useDisclosure();
+
+  return (
+    <Box
+      maxW="sm"
+      mb="24px"
+      border="1px solid"
+      borderColor="hubbard.100"
+      borderRadius="8px"
+    >
+      <Box p="6">
+        <HStack spacing="112">
+          <Text mb="16px" textStyle="mobileBodyBold">
+            {startDate}
+          </Text>
+          <Menu isLazy>
+            <MenuButton
+              as={IconButton}
+              aria-label="options"
+              icon={<EllipsisIcon />}
+              variant="ghost"
+            />
+            <MenuList p={0} minW="0" w="94px">
+              <MenuItem
+                onClick={() =>
+                  history.push(
+                    `/dropoff/${schedule.id}?donorId=${schedule.donorId}`,
```

Comment:
I dont think we should pass in `donorId` as a query param here. In `EditDropoffPage` component, we are getting the current schedule. That has the `donorId`, we should use that.